### PR TITLE
Minor spelling changes in misc comments.

### PR DESCRIPTION
**Pull request by eerwitt**
None of these spelling changes seem important but I came across them while understanding how the project works and thought I'd send a pull request.

### CODE REVIEW

File: tensorflow_serving/util/optional.h [hunk 1]
```diff
@@ -34,7 +34,7 @@ namespace serving {
 // an "empty" value.  When it holds a value of T, it stores it as a direct
 // subobject, so sizeof(optional<T>) is approximately sizeof(T)+1. The interface
 // is based on the upcoming std::experimental::optional<T>, and
-// tensorflow::serving::optional<T> is designed to be cheaply drop-in replacable
+// tensorflow::serving::optional<T> is designed to be cheaply drop-in replaceable
```
Comment (kirilg) [inline tensorflow_serving/util/optional.h]: Over 80 characters, please move "replaceable" to next line.

File: tensorflow_serving/util/optional.h [hunk 2]
```diff
@@ -366,7 +366,7 @@ class optional {
   // Whether or not this optional is non-empty.
   bool engaged_ = false;
 
-  // T constaint checks.  You can't have an optional of nullopt_t, in_place_t or
+  // T constraint checks.  You can't have an optional of nullopt_t, in_place_t or
```
Comment (kirilg) [inline tensorflow_serving/util/optional.h]: This also goes over 80 chars, maybe just change the double space to one.

**Comment by kirilg** [comment]
Thanks for this change Erik! Looks good, just some minor formatting nits.

**Comment by eerwitt** [comment]
Sorry about that, I've updated my editor to reflect.

Would you like me to squash the commits too?

**Comment by kirilg** [comment]
Yes, please squash the commits.

**Comment by eerwitt** [comment]
Squashed. Thank you for this project, it's a joy to work with so far.

**Pull request merged**